mozregression/launchers.py

```diff
@@ -275,9 +275,58 @@ def create_launcher(buildinfo):
     )
 
 
+class FirefoxRegressionProfile(Profile):
+    """Specialized Profile subclass for Firefox"""
+
```

Comment:
I guess we can start from this.
People use mozregression for test purpose so I kept most of the preferences.
Since toolkit.telemetry.enabled is set False it looks fine to remove server config.